### ISSUE TITLE
'Disable user' feature

### ISSUE THREAD

**Issue by okms** [Puzzlepart/did#548]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
Issue #545 introduced the option of deleting a user through the admin ui, likewise we should be able to disable a user

### Description
When a user is on leave or for other reasons are not an active employee, we should allow admins to disable this user in Did. The effect of this would be to hide them from reports and the overview pages. This would in effect be a soft delete. 

A more advanced implementation would be to show the employee hours in older reports, but hide them from the time of deactivation. 

As such the flag in Azure table store could be 
- A bool ("disabled")
- A date ("disabled_date")

If we have both we can potentially implement the advanced case, even if we go for the simpler implementation first
